package.json

```diff
@@ -28,7 +28,7 @@
     "graphql": "^16.8.0",
     "js-cookie": "^3.0.5",
     "lodash": "^4.17.21",
-    "next": "^13.4.19",
```

Comment:
based on it being the only change in this PR, I take it this update is seamless?
nothing breaking sees surprising for a major version update.

Reply:
Yessir. checked the breaking changes in the changelog and verified locally